ismatec/driver.py

```diff
@@ -97,24 +96,25 @@ async def set_mode(self, channel: int, mode: Protocol.Mode):
         assert channel in self.channels
         return self.hw.command(f'{channel}{mode.value}')
 
-    def get_number_channels(self) -> int:
-        """Get the number of (currently configured) pump channels.
+    def get_channels(self) -> list:
```

Comment:
It seems like this function is only called once and to immediately make a list of available channel options. Felt cleaner to me to put that post-processing in the method.